nuklear.h

```diff
@@ -480,7 +480,8 @@ struct nk_rect {float x,y,w,h;};
 struct nk_recti {short x,y,w,h;};
 typedef char nk_glyph[NK_UTF_SIZE];
 typedef union {void *ptr; int id;} nk_handle;
-struct nk_image {nk_handle handle;unsigned short w,h;unsigned short region[4];};
+struct nk_image {nk_handle handle; nk_ushort w, h; nk_ushort region[4];};
+struct nk_9slice {struct nk_image img; nk_ushort l, t, r, b;};
```

Comment:
The naming here might be problematic for other language bindings. In those the `nk_` prefix is usually removed and moved to e.g. package name (or whatever target language provides), so in those bindings we got something like `nk.image` instead of `nk_image`. I'm author of Common Lisp bindings, and there probably wouldn't be the problem there - CL allows for names like `9slice`, but other languages might not.

Reply:
Maybe rename to [`scale9` or `grid9` or `slice9`](https://en.wikipedia.org/wiki/9-slice_scaling )?

Reply:
Something like `nine_slice` might do the trick as well.

Reply:
Renamed 9slice to nine_slice
Let me know if you need anything else :slightly_smiling_face: